### PR TITLE
Issue #430 by Inlead: Modify existing PR regarding Search API.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/430

### CODE REVIEW

File: themes/ddbasic/template.php [hunk 1]
```diff
@@ -397,6 +397,25 @@ function ddbasic_preprocess_node(&$variables, $hook) {
...
+            'html' => FALSE,
+          ),
+          '#prefix' => '<span class="page-link">',
+          '#surfix' => '</span>',
+          '#weight' => 6,
+        );
+
+        $variables['content']['group_right_col_search']['more_link'] = $more_link;
+        break;
     }
```
Comment (holt83) [inline themes/ddbasic/template.php]: This is a small one, but you should never use theme link directly. Some preprocessing from the l() function gets lost this way. If you want to output a link with render-array, you should use '#type' => 'link', which adds a pre-render callback that takes care of calling l().

More info: http://drupal.stackexchange.com/questions/24691/how-to-theme-a-link-in-a-d7-render-array#answer-131855

Comment (JrgenGNielsen) [inline themes/ddbasic/template.php]: @inlead : I second that

Comment (guddo) [inline themes/ddbasic/template.php]: We have tried this without success.

Comment (JrgenGNielsen) [inline themes/ddbasic/template.php]: keys are a bit different - what you need is something like this;
`   $link = array(
    '#type'  => 'link',
    '#title' => 'foo link',
    '#href'  => 'node/foo',
    '#options' => array(
      'attributes' => array(
        'title' => 'foo title',
      ),
    ),
    '#prefix' => '<p>',
    '#suffix' => '</p>'
  );`

Comment (guddo) [inline themes/ddbasic/template.php]: https://github.com/ding2/ding2/pull/194/commits/f0e08ad9c80a960a0b6529039a38e09443530509

**Comment by JrgenGNielsen** [comment]
reviewed and accepted

File: themes/ddbasic/template.php (hunk 1 above)
Comment (cableman) [inline themes/ddbasic/template.php]: The link theme function do not work if the attributes class value is not set. But as written in this ticket in redmind: I have pull in this PR and the other one an working on an new PR with search API.

**Comment by guddo** [comment]
@cableman Does #236 duplicate this? How do you imagine us to rebase this?

**Comment by guddo** [comment]
Bump @cableman

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 1165cedfbc0b00455cf5bcdc622298eca62c6afe
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 1165cedfbc0b00455cf5bcdc622298eca62c6afe
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Pull request merged**